Dockerfile

```diff
@@ -1,12 +1,8 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
 MAINTAINER "Francesco Romani" <fromani@redhat.com>
 ENV container docker
 
-RUN yum -y update
```

Comment:
these images doesn't need update?

Reply:
Not sure. AFAIU they are supposed to always be up to date. Anyway, the update mechanism on -minimal is going to be different, so this should be changed anyway.
I plan to drop the update and to reintroduce it later if needed.

Reply:
According to that article, it should be ok to drop update command.

Reply:
@ksimon1 worst case I'll reintroduce it again later :)